### ISSUE TITLE
Ecommerce tracking fails

### ISSUE THREAD

**Issue by Kesquom** [piwik/piwik#8398]
Since 2.13 I don't get any ecommerce tracking data. Standard website tracking works fine (visits, links...) but the specific product data, order data etc. is missing

I Repoduced the error on a test system (php 5.6.5): 

1. Fresh Piwik 2.14.1 installation with new database and default settings (no IP anonymization, ignore do-not-track, currency: Euro) -> no ecommerce data
2. Fresh Piwik 2.12 installation with new database and default settings (no IP anonymization, ignore do-not-track, currency: Euro) -> shows ecommerce data (products, skus, prices ...)
3. Piwik 2.12 to 2.14.1 upgrade -> no new ecommerce data

So basically this code seems to be getting ignored somehow:

```javascript
_paq.push(['addEcommerceItem',
"12345",
"example product",
["category"],
123.0000]);
```

(my first bug report on github, please be patient with me)

**Comment by Glisse1** [comment]
After upgrading piwik to 2.14 I also lost part of the ecommerce tracking (specifically cart updates) using woocommerce.. no new cart updates with products, skus, etc unless the customer reached the cart page..but the order itself was fine, recorded in thank you page as it should.. I had to analyze and "hack" the wordpress piwik integration plugin that i was using, basically took the code running on cart page and duplicated it on "add to cart button" for cart updates... and now everything it is fine again.

**Comment by mattab** [comment]
@Glisse1 were you using the latest version of Woocommerce plugin? if you have a problem with it, please make sure to create a bug report in: https://wordpress.org/support/plugin/woocommerce-piwik-integration

**Comment by mattab** [comment]
Hi @Kesquom 

can you send us a URL to your website URL and i'll try to reproduce the `addEcommerceItem` workflow. it may help if you enable the tracker debugging as explained in: http://developer.piwik.org/api-reference/tracking-api#debugging-the-tracker
